### DISCUSSION TITLE
Free energy difference as a function of time

### DISCUSSION THREAD

**Discussion by pallavsen007** [alchemistry/alchemlyb#423] [Q&A]
Time (ps) Forward (kJ/mol) Time (ps) Reverse (kJ/mol)
50.0 28.558 +- 1.766 0.0 20.017 +- 0.608
100.0 23.895 +- 1.474 50.0 19.122 +- 0.645
150.0 21.008 +- 1.144 100.0 19.478 +- 0.675
200.0 20.907 +- 0.991 150.0 19.671 +- 0.732
250.0 20.431 +- 0.877 200.0 19.919 +- 0.781
300.0 20.385 +- 0.789 250.0 20.333 +- 0.877
350.0 19.907 +- 0.731 300.0 20.232 +- 1.013
400.0 20.133 +- 0.684 350.0 20.306 +- 1.189
450.0 20.475 +- 0.641 400.0 19.996 +- 1.460
500.0 20.017 +- 0.608 450.0 17.342 +- 1.932

Data from samples provided in alchemical-analysis repo

My question is If i am using BAR and have various windows of same lengths how do I get a similar output?

**Comment by xiki-tempula** [comment]
Does this help? https://alchemlyb.readthedocs.io/en/latest/convergence.html

**Comment by pallavsen007** [comment]
I do not get what I am plotting for first

**Comment by pallavsen007** [comment]
what is the difference between plotting probability distributions of potential energy and this
